content/posts/newsletter/2020-04-02.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Draft Title - Date
+save_as: tba
+url: tba
+date: 2020-04-02 23:40
+author: emus
+blog_id: newsletter
+category: Newsletter
+
+---
+
+# Collaboration over the Internet! XX May 2020
+
+## Intro
+
+Welcome to the XMPP newsletter covering the month of April 2020.
+
+We are always happy for contributors - just come along and talk to us in the [Comm-Team MUC](xmpp:commteam@muc.xmpp.org?join) and thereby help us sustain this as a community effort, which [process is fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+
+[Subscribe](https://tinyletter.com/xmpp) to receive the next edition in your inbox!
+
+
+#### Newsletter translations
+
+The translations of the XMPP Newsletter will be release here:
+
+- [https://www.jabber.de/ German]
+- [https://news.jabberfr.org/category/newsletter/ French]
+- [https://www.jabber.de/ Spanish]
+
+
+## XMPP Standards Foundation
+
+The XSF is renewing their sponsorship for 2020. Many thanks for everyone supporting and especially sponsoring in 2019. We would be happy if your organization would consider [sponsoring also in 2020](https://mail.jabber.org/pipermail/standards/2020-April/037254.html)! If your organization considers to sponsor the first time, [our mission may interests you as well](https://xmpp.org/about/xsf/mission).
+
+Alexander Gnauck published the [Membership Applications Q2 2020](https://wiki.xmpp.org/web/Membership_Applications_Q2_2020). Applications are welcome!
+
+
+## Articles
+
+[Pasis](https://github.com/pasis), maintainer of libstrophe and contributor to profanity, wrote about using the [XML console in profanity](https://profanity-im.github.io/blog/post/xml-console-in-profanity/) for debugging.
+
+Jan Cieśla from MongooseIM wrote a blog post how [they thrive their project by the metrics](https://www.erlang-solutions.com/blog/how-data-drives-mongooseim.html).
+
+Can you believe? Zoom uses XMPP for its chat - cool! [Vulnerability](https://blog.talosintelligence.com/2020/04/zoom-user-enumeration.html) - not cool.
+
+## Software releases
+
+Dino has now [Last Message Correction](https://github.com/dino/dino/commit/871ff33ac79f3d17b0260b8bfcd27780038edd6d) for all the typos and regrets you made. And also [voice handleing in moderated groups](https://github.com/dino/dino/commit/2631a9bdbaf9a40f329f05c55c6e2ea38efeb10c)
```

Comment:
typos and regrets you made => typos you made and regrets you have
voice handleing => voice handling